<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>水声脉冲信号侦获软件界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0">
           <item>
            <widget class="QLabel" name="title">
             <property name="text">
              <string>水声脉冲信号侦获软件</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>345</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="minButton">
             <property name="text">
              <string>min</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="closeButton">
             <property name="text">
              <string>close</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="maxButton">
             <property name="text">
              <string>max</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1,1,1,8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
               <item>
                <widget class="QPushButton" name="model_choose">
                 <property name="text">
                  <string>model</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="model_path">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1,1,1,1">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>input</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="img_choose">
                 <property name="text">
                  <string>img</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="imgs_choose">
                 <property name="text">
                  <string>imgs</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="wav_choose">
                 <property name="text">
                  <string>wav</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="realtime_chose">
                 <property name="text">
                  <string>realtime</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>inputwavpath</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="inputwavpath">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>IOU</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="iouSpinBox">
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.450000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>conf</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="confSpinBox">
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.250000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0,1">
               <item>
                <widget class="QCheckBox" name="autosave_checkBox">
                 <property name="text">
                  <string>auto_save</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="outputpath_choose">
                 <property name="text">
                  <string>outputpath</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="output_path">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>STFT</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="stftfs">
                  <property name="text">
                   <string>fs/Hz</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="stftfsSpinBox">
                  <property name="maximum">
                   <number>100000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="wlen">
                  <property name="text">
                   <string>wlen</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="wlenspinBox">
                  <property name="maximum">
                   <number>10000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>result</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QListWidget" name="result_listWidget"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="0,9">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,0,0">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>View</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="start">
                 <property name="text">
                  <string>start</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="stop">
                 <property name="text">
                  <string>stop</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="save">
                 <property name="text">
                  <string>save</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSplitter" name="splitter">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QGraphicsView" name="raw_img"/>
               <widget class="QGraphicsView" name="detect_img"/>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
